ExternalModule.php

```diff
@@ -9,11 +9,42 @@
 use ExternalModules\AbstractExternalModule;
 use ExternalModules\ExternalModules;
 use REDCap;
+use Logging;
 
 /**
  * ExternalModule class for SuspensionWarning.
  */
 class ExternalModule extends AbstractExternalModule {
+ 	/**
+     * @inheritdoc
+     */
+    function redcap_every_page_before_render($project_id) 
+    {
+    	$base_page = 'redcap/?username=';
```

Comment:
This is not written portably. Many sites do not include "/redcap/" in their REDCap base url.